libs/langchain/langchain/llms/ai21.py

```diff
@@ -62,7 +62,7 @@ class AI21(LLM):
     logitBias: Optional[Dict[str, float]] = None
     """Adjust the probability of specific tokens being generated."""
 
-    ai21_api_key: Optional[str] = None
+    ai21_api_key: Optional[SecretStr] = None
```

Comment:
Is it possible for the ai21_api_key to be None? It looks like after the root validator runs this always get set to a value?

Reply:
Yes, ai21_api_key can never be None.